### PR TITLE
fixing bug with data description

### PR DESCRIPTION
**Pull request by dyf**

### CODE REVIEW

File: src/aind_data_schema/__init__.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 """ imports for AindModel subclasses
 """
 
-__version__ = "0.3.10"
+__version__ = "0.3.11"
```
Comment (jtyoung84) [inline src/aind_data_schema/__init__.py:4]: The version should be managed automatically now

Comment (dyf) [reply src/aind_data_schema/__init__.py:4]: Jon so I undo the version change?

**Review by jtyoung84 — CHANGES_REQUESTED** [review]
No need for manual version bump

File: src/aind_data_schema/__init__.py (hunk 1 above)
Comment (jtyoung84) [reply src/aind_data_schema/__init__.py:4]: yeah

Comment (dyf) [reply src/aind_data_schema/__init__.py]: (done)

**Review by jtyoung84 — APPROVED** [review]

**Pull request merged**